pkg/storaged/resize.jsx

```diff
@@ -241,22 +289,22 @@ export function grow_dialog(client, lvol, info, to_fit) {
         passphrase_fields = existing_passphrase_fields(_("Resizing an encrypted filesystem requires unlocking the disk. Please provide a current disk passphrase."));
 
     if (!usage.Teardown && size_fields.length + passphrase_fields.length === 0) {
-        return lvol_and_fsys_resize(client, lvol, grow_size, info.grow_needs_unmount, null);
+        return lvol_or_part_and_fsys_resize(client, lvol_or_part, grow_size, info.grow_needs_unmount, null);
     }
 
     const dlg = dialog_open({
-        Title: _("Grow logical volume"),
+        Title: title,
         Teardown: TeardownMessage(usage),
         Fields: size_fields.concat(passphrase_fields),
         Action: {
             Title: _("Grow"),
             action: function (vals) {
                 return teardown_active_usage(client, usage)
                         .then(function () {
-                            return (lvol_and_fsys_resize(client, lvol,
-                                                         to_fit ? grow_size : vals.size,
-                                                         info.grow_needs_unmount,
-                                                         vals.passphrase || recovered_passphrase)
+                            return (lvol_or_part_and_fsys_resize(client, lvol_or_part,
+                                                                 to_fit ? grow_size : vals.size,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19385-20230928-071939-cd48ea23-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)